prime-router/metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform.yml

```diff
@@ -276,3 +277,48 @@ elements:
     resource: '%diagnosticReport.basedOn.resolve().requester.resolve().organization.resolve().telecom.where(system = "phone")'
     resourceIndex: telecomIndex
     schema: datatypes/simple-report-telecom
+
+  # populate Specimen identifier
+  - name: sr-specimen-placer-assigned-identifier-value
+    bundleProperty: '%specimen.identifier[0].value'
```

Comment:
Just to be safe, I would maybe moved `%specimen.identifier[0]` to be the resource and then have a condition to check for it?

You could also consider breaking this into a separate file, but up to you

Reply:
I'm actually overwriting `%specimen.identifier[0]` on purpose.

In the example Simple Report FHIR files it is:
```
"identifier" : [
          {
            "value" : "5f0d29c2-4181-4a1e-8774-b88338602681"
          }
        ],
```

But that value never shows up in the expected HL7. Instead we see the `diagnosticReport.id`.

Reply:
ah ok... yea even more reason to try to distill what's going on here in a comment.

Reply:
can do